server/src/main/resources/org/elasticsearch/TransportVersions.csv

```diff
@@ -131,3 +131,4 @@
 8.15.1,8702002
 8.15.2,8702003
 8.15.3,8702003
+8.15.4,8702003
```

Comment:
Probably not strictly necessary for the tests for pass. But it would be rather odd if we update `IndexVersions.csv` without the corresponding change here.